libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
1. No. The warning that can cause a crash is not a good warning.
2. Yes, it is certainly difficult. Which is why I suggested making it **explicitly** clear that the two PRs conflict with each other, and if both are to be accepted - one would have to change.

When I was merging PRs from vlm's repo, if a PR caused a conflict - I either skipped it altogether, or asked the author to rewrite it so that it would go in smoothly. Same situation here. I should've asked you to rewrite #23. I didn't. I know better now. ;-)